frontend/src/routes/Infrastructure/Automations/AnsibleAutomationsForm.tsx

```diff
@@ -588,21 +619,45 @@ function EditAnsibleJobModal(props: {
             onClose={() => props.setAnsibleJob()}
         >
             <AcmForm>
+                <FormGroup fieldId="radio" isInline>
+                    <Radio
+                        name={'job-template'}
+                        id={'job-template'}
+                        label={t('Job template')}
+                        isChecked={filterForJobTemplates}
+                        onChange={() => {
+                            setFilterForJobTemplates(true)
+                            clearTemplateName()
+                        }}
+                    />
+                    <Radio
+                        name={'workflow-template'}
+                        id={'workflow-template'}
+                        label={t('Workflow job template')}
+                        isChecked={!filterForJobTemplates}
+                        onChange={() => {
+                            setFilterForJobTemplates(false)
+                            clearTemplateName()
+                        }}
+                    />
+                </FormGroup>
                 <AcmSelect
                     maxHeight="18em"
                     menuAppendTo="parent"
-                    label={t('template.modal.name.label')}
+                    label={t(
+                        filterForJobTemplates ? 'template.modal.name.label' : 'template.workflow.modal.name.label'
+                    )}
                     id="job-name"
                     value={ansibleJob?.name}
                     onChange={(name) => {
-                        if (ansibleJob) {
-                            const copy = { ...ansibleJob }
-                            copy.name = name as string
-                            setAnsibleJob(copy)
-                        }
+                        newTemplateSelection(name)
                     }}
                     variant={SelectVariant.typeahead}
-                    placeholder={t('template.modal.name.placeholder')}
+                    placeholder={t(
+                        filterForJobTemplates
+                            ? 'template.modal.name.placeholder'
```

Comment:
Move the function call inside the ternary. `t` should always be called with a string literal.